src/main/java/org/concordion/Concordion.java

```diff
@@ -117,7 +118,7 @@ public void finish() {
 
 
     private String createMultipleSpecsMessage(Fixture fixture, SpecificationType type1, SpecificationType type2) {
-        return String.format("Found multiple matching specifications: '%s.%s' and '%1$s.%s'",
+        return SimpleFormatter.format("Found multiple matching specifications: '%s.%s' and '%1$s.%s'",

```

Comment:
I just noticed this is using %1$s to reuse the first parameter. I don't think SimpleFormatter handles that. I'd be OK if you repeated the first parameter instead (preferably extracted to a local variable)